Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceManagmentAndroidDeviceOwnerEnrollmentProfile/MSFT_IntuneDeviceManagmentAndroidDeviceOwnerEnrollmentProfile.schema.mof

```diff
@@ -0,0 +1,35 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneDeviceManagmentAndroidDeviceOwnerEnrollmentProfile")]
+class MSFT_IntuneDeviceManagmentAndroidDeviceOwnerEnrollmentProfile : OMI_BaseResource
+{
+    [Key, Description("Display name for the enrollment profile.")] String DisplayName;
+    [Write, Description("Unique GUID for the enrollment profile. Read-Only.)] String Id;
+
+    [Write, Description("Intune AccountId GUID the enrollment profile belongs to.")] String AccountId;
+    [Write, Description("Description for the enrollment profile.")] String Description;
+    [Write, Description("The enrollment mode of devices that use this enrollment profile."), ValueMap{"corporateOwnedDedicatedDevice", "corporateOwnedFullyManaged", "corporateOwnedWorkProfile", "corporateOwnedAOSPUserlessDevice", "corporateOwnedAOSPUserAssociatedDevice"}] String EnrollmentMode;
+    [Write, Description("The enrollment token type for an enrollment profile."), ValueMap{"default", "corporateOwnedDedicatedDeviceWithAzureADSharedMode", "deviceStaging"}] String EnrollmentTokenType;
+    [Write, Description("Date time the enrollment profile was created.")] DateTime CreatedDateTime;
+    [Write, Description("Date time the enrollment profile was last modified.")] DateTime LastModifiedDateTime;
+    [Write, Description("Value of the most recently created token for this enrollment profile.")] String TokenValue;
+    [Write, Description("Date time the most recently created token was created.")] DateTime TokenCreationDateTime;
+    [Write, Description("Date time the most recently created token will expire.")] DateTime TokenExpirationDateTime;
```

Comment:
DateTime is unfortunately not a valid MOF parameter. Instead, please declare those as String.